backend/src/main/java/com/sim_backend/rest/model/BootNotification.java

```diff
@@ -0,0 +1,73 @@
+package com.sim_backend.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.annotations.SerializedName;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+/** A OCPP Boot Notification Request Message. */
+@Getter
+@SuperBuilder
+@OCPPMessageInfo(messageCallID = OCPPMessage.CALL_ID_REQUEST, messageName = "BootNotification")
+public final class BootNotification extends OCPPMessage {
+
+  /** The Charge Point's Vendor. */
+  @SerializedName("chargePointVendor")
+  private final String chargePointVendor;
+
+  /** The Charge Point's Model. */
+  @SerializedName("chargePointModel")
+  private final String chargePointModel;
+
+  /** The Charge Point's Serial Number. */
+  @SerializedName("chargePointSerialNumber")
+  private final String chargePointSerialNumber;
+
+  /** The Charge Box's Serial Number. */
+  @SerializedName("chargeBoxSerialNumber")
+  private final String chargeBoxSerialNumber;
+
+  /** Firmware Version. */
+  @SerializedName("firmwareVersion")
+  private final String firmwareVersion;
+
+  /** ICCID (Integrated Circuit Card Identifier) . */
+  @SerializedName("iccid")
+  private final String iccid;
+
+  /** IMSI (International Mobile Subscriber Identity). */
+  @SerializedName("imsi")
+  private final String imsi;
+
+  /** Meter Type. */
+  @SerializedName("meterType")
+  private final String meterType;
+
+  /** Meter Serial Number. */
+  @SerializedName("meterSerialNumber")
+  private final String meterSerialNumber;
+
+  // Jackson annotations for deserialization
+  @JsonCreator
```

Comment:
This doesn't take effect because we are using gson for serialization.

Reply:
Removed